content/en/docs/concepts/policy/pod-security-standards.md

```diff
@@ -0,0 +1,323 @@
+---
+reviewers:
+- tallclair
+title: Pod Security Standards
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Security settings for pods are typically applied by using [security
+contexts](../../tasks/configure-pod-container/security-context/). Security Contexts allows for the
+definition of privilege and access controls on a per-pod basis.
+
+The enforcement and policy-based definition of cluster requirements of security contexts has
+previously been achieved using [Pod Security Policy](pod-security-policy/). A _Pod Security Policy_
+is a cluster-level resource that controls security sensitive aspects of the pod specification.
+
+However, numerous means of policy enforcement have arisen that augment or replace the use of
+PodSecurityPolicy. The intent of this page is to detail recommended pod security profiles, decoupled
+from any specific instantiation.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Policy Types
+
+There is an immediate need for base policy definitions to broadly cover the security spectrum. These
+should range from highly restricted to highly flexible:
+
+- **_Privileged_** - Unrestricted policy, providing the widest possible level of permissions. This
+  policy allows for known privilege escalations.
+- **_Baseline/Default_** - Minimally restrictive policy while preventing known privilege
+  escalations. Allows the default (minimally specified) pod configuration.
+- **_Restricted_** - Heavily restricted policy, following current pod hardening best practices.
+
+## Policies
+
+### Privileged
+
+The Privileged policy is purposely-open, and entirely unrestricted. This type of policy is typically
+aimed at system- and infrastructure-level workloads managed by privileged, trusted users.
+
+The privileged policy is defined by an absence of restrictions. For blacklist-oriented enforcement
+mechanisms (such as gatekeeper), the privileged profile may be an absence of applied constraints
+rather than an instantiated policy. In contrast, for a whitelist oriented mechanism (such as Pod
+Security Policy) the privileged policy should enable all controls (disable all restrictions).
+
+### Baseline/Default
+
+The Baseline/Default policy is aimed at ease of adoption for common containerized workloads while
+preventing known privilege escalations. This policy is targeted at application operators and
+developers of non-critical applications. The following listed controls should be
+enforced/disallowed:
+
+<table>
```

Comment:
Can you use a Markdown table here? If you can, also add a [caption](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#table-captions) using a shortcode; if not, please add an equivalent caption manually.

Reply:
Unfortunately not, because of the nested lists (at least I couldn't figure out how to make it work). I copied the caption HTML from the rendered table in the style guide.